internal/build/pom.xml

```diff
@@ -1054,5 +1057,21 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <!-- Profile to skip time-consuming steps. -->
+            <id>fast</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
```

Comment:
this is not necessary. And `activeByDefault` is somewhat funky because marking a profiles as "active as default true" actually disables all other profiles.

Reply:
True that the default is false
`<xs:element minOccurs="0" name="activeByDefault" type="xs:boolean" default="false"></xs:element>`

> [..] `activeByDefault` is somewhat funky because marking a profiles as "active as default true" actually disables all other profiles

The documentation describes it differently: If set to true, this profile will be active unless another profile in this pom is activated using the command line -P option or by one of that profile's activators.

Reply:
as I said, it is funky. :-)